src/utils/init.ts

```diff
@@ -174,25 +178,20 @@ async function updateUserId(newUserId: string, merge = false): Promise<void> {
   }
 }
 
-function _parseUserVal(userVal: FeatheryFieldTypes, key: string) {
-  // TODO(ts): Should we make an internal vs external FeatheryFieldTypes? or just | File like this?
-  let val: FeatheryFieldTypes | File = userVal;
-  if (isBase64Image(val)) val = dataURLToFile(val, `${key}.png`);
-  // If the value is a file type, convert the file or files (if repeated) to Promises
-  return val instanceof File ? Promise.resolve(val) : val;
-}
-
 /**
  * If customers provide files through setFieldValues
  * we need to explicitly convert any files to file Promises
  * since they may not have done so
  */
 function setFieldValues(userVals: FieldValues, rerender = true): void {
+  console.warn(
```

Comment:
isn't this still undeprecated at the global level?

Reply:
Yes. Good catch.  Forgot to remove it there.